source/documentation/06-api-reference.md

```diff
@@ -134,6 +136,34 @@ These error codes provide more information about why a request failed.
 | General | P0920 | Request blocked by security rules | Our firewall blocked your request. See Troubleshooting section for details. |
 | General | P0999 | GOV.UK Pay is unavailable | The GOV.UK Pay service is temporarily down. |
 
+### API errors caused by payment statuses
+
+The API error codes in this section are driven by payment status. You can see which payments have these error codes using either of the following methods:
+
+- log into the [GOV.UK Pay admin site](https://selfservice.payments.service.gov.uk/), go to _Transactions_ and click on a failed payment
```

Comment:
(This is actually unchanged from the first commit but I only just noticed it)

We have “failed” as one of our external payment states (alongside “created”, “started”, “submitted”, “success”, “cancelled” and “error”). The error codes listed here could belong to “failed” payments but also “cancelled” and “error” payments. So perhaps we should say “click on an unsuccessful payment” or something.